src/apps/opening-hours-sidebar/OpeningHoursSidebarButtonLarge.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { FC } from "react";
+import iconWatch from "@danskernesdigitalebibliotek/dpl-design-system/build/icons/basic/icon-watch-static.svg";
+import { useText } from "../../core/utils/text";
+
+type ButtonLargeProps = {
+  openDialog: () => void;
+};
+
+const ButtonLarge: FC<ButtonLargeProps> = ({ openDialog }) => {
```

Comment:
What is the difference in usage between the small and big button? 
Would it make sense to make a story with one and another with the other?